db/migrate/20200510164927_add_amount_to_item.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Migration to add amount of the item that was initially at payment_intent item
+class AddAmountToItem < ActiveRecord::Migration[6.0]
```

Comment:
Why are we adding amount to Item? Amount is variable, and is derived from the GiftCardAmount on the DonationDetail

Reply:
I thought that since GiftCardAmount would be used to track the balance, once we enable the gift card redemption and we start decrementing, it might get weird